submissions/modupeAdeOnojobi/html-forms/contact.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact</title>
+</head>
+<body>
+    <h1>Sneak Nation</h1>
+    <nav>
+        <ul>
+            <li><a href="../html/index.html">Home</a></li>
+            <li><a href="../html/index.html">Product</a></li>
```

Comment:
I think you should remove the link from Home, since the page is for products. Also, you can rename to products.html or productPage.html in place of index.